unitTests/ui/NoNameFacet/NoNameFacetHeaderTest.ts

```diff
@@ -79,5 +79,46 @@ export function NoNameFacetHeaderTest() {
 
       expect($$(operatorElement).findClass('coveo-or')).toBeTruthy();
     });
+
+    it(`when clicking on the toggle Operator button
+      should switch the option correctly`, () => {
+      baseOptions.rootFacetOptions.enableTogglingOperator = true;
```

Comment:
Similar to the comment above, what do you think of the name `enableOperator`?

Reply:
Maybe it could work, but in this case it's different than the actual name

Reply:
Indeed ... okay, in that case I think it would be better to keep it as-is and reuse the existing name. Should a client want to use the new facet, they will only need to change the class name (e.g. CoveoFacet -> CoveoIntelligentFacet) without having to change the option names.

Reply:
It is not that important, I dont think just changing we have to constraint ourselves too much with the older facets

Reply:
Erhmmm...

For this specific option, I would remove it, as explained with my other comment.

In general though, I would very much try to keep the exact same name/option when applicable. 


People are used to existing option name, and changing "just because" would understandably be very annoying from their point of view...